docs/Other SDL Features/Getting In-Car Microphone Audio/index.md

```diff
@@ -209,4 +208,4 @@ performAPT.setOnRPCResponseListener(new OnRPCResponseListener() {
     }
 });
 ```
-!@
\ No newline at end of file
+!@
```

Comment:
I'm confused as to what is happening here, why is this in this PR?

Reply:
you mean the removal of the ###### Ford HMI or line 211?

Reply:
Both

Reply:
oh my bad I committed this to the wrong branch. I will remove it.